src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -60,10 +60,33 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
         }
 
         this._stripeUPEClient = await this._loadStripeJs(stripePublishableKey, stripeConnectedAccount);
+        let appearance: StripeUPEAppearanceOptions | undefined;
+        if (stripeupe.style) {
+            const styles = stripeupe.style;
+            appearance = {
+                variables: {
+                    colorPrimary: styles.fieldInnerShadow,
+                    colorBackground: styles.fieldBackground,
+                    colorText: styles.labelText,
+                    colorDanger: styles.fieldErrorText,
+                    colorTextSecondary: styles.labelText,
+                    colorTextPlaceholder: styles.fieldPlaceholderText,
+                    colorIcon: styles.fieldPlaceholderText,
+                },
+                rules: {
+                    '.Input': {
+                        borderColor: styles.fieldBorder,
+                        color: styles.fieldText,
+                        boxShadow: styles.fieldInnerShadow,
+                    },
+                },
+            };
+        }
 
         this._stripeElements = this._stripeElements ?? this._stripeUPEClient.elements({
             clientSecret: paymentMethod.clientToken,
             locale: formatLocale(shopperLanguage),
+            appearance,
```

Comment:
Does the API accept `undefined`? Since if styles are not passed then this value will be undefined.

Reply:
Yes
appearance undefined
![image](https://user-images.githubusercontent.com/87149598/162330631-76582bc6-16f3-4a73-bdd9-45eec1d42880.png)
![image](https://user-images.githubusercontent.com/87149598/162330655-0ba10b1e-7223-4d2e-84b3-ca4d26369da8.png)

Property undefined
![image](https://user-images.githubusercontent.com/87149598/162330945-e8bacda3-7cf7-4dae-9c33-1271791ce67e.png)
![image](https://user-images.githubusercontent.com/87149598/162330980-75fa333b-6ff1-4ca6-8149-68c33b746331.png)